src/django/api/reports/facilities_marked_closed.sql

```diff
@@ -0,0 +1,4 @@
+SELECT to_char(approved_at, 'YYYY-MM') as month, COUNT(*) as facilities_closed_in_month
+FROM api_HistoricalFacilityActivityReport
+WHERE status = 'CONFIRMED'
+GROUP BY month;
```

Comment:
We should include `ORDER BY month` so this is sorted like our other reports.

Reply:
Updated in 44a9a1f